Form/Type/ChangePasswordFormType.php

```diff
@@ -37,12 +37,21 @@ public function __construct($class)
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $constraints = array(
```

Comment:
this is ``$constraintOptions``, not ``$constraints``. It is not an array of constraints.